components/util/opts.rs

```diff
@@ -361,6 +368,7 @@ pub fn print_debug_usage(app: &str) -> ! {
     print_option("dump-display-list", "Print the display list after each layout.");
     print_option("dump-display-list-json", "Print the display list in JSON form.");
     print_option("dump-layer-tree", "Print the layer tree whenever it changes.");
+    print_option("record", "Serializes all frame data sent to webrender and writes to record folder.");
```

Comment:
Is "frame data" == display list? Just say "Serialize the display list ..." sounds clearer to me.